.gitignore

```diff
@@ -0,0 +1,25 @@
+# Android Studio
+*.iml
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+
+# Mac generated files
+.DS_Store
+
+# Build files
+/build
+/captures
+
+# External native build folder generated in Android Studio 2.2 and later
+.externalNativeBuild
+.cxx
+
+# Gradle
+.gradle
+
+# Local configs
+local.properties
```

Comment:
please add Android Navigation component in build.gradle
exclude it from git 
# Android Studio Navigation editor temp files
.navigation/

Reply:
# Log Files
*.log

Reply:
# Intellij
*.iml
.idea/*